draft-ietf-quic-transport.md

```diff
@@ -137,6 +137,9 @@ protocol, including the conceptual design, wire format, and mechanisms of the
 QUIC protocol for connection establishment, stream multiplexing, stream and
 connection-level flow control, connection migration, and data reliability.
 
+QUIC used to be known as 'Quick UDP  Internet Connections', but the present
+document advises it be used no  more: QUIC does not stand for anything.
+
 Accompanying documents describe QUIC's loss detection and congestion control
```

Comment:
I belive the feedback on the mailing list was that the RFC is not the place to make historical references.

>QUIC used to be known as 'Quick UDP  Internet Connections', but the present
 +document advises it be used no  more: QUIC does not stand for anything.

could perhaps be reformulated as

> QUIC is a name and not an acronym.

Reply:
Fixed in the new pull, @mikkelfj. Have you seen it?

Reply:
now I have :)